### ISSUE TITLE
Training images

### ISSUE THREAD

**Issue by joyJZhang** [jonrein/tensorflow_CTC_example#7]
hello,i used your model to train myself captcha images,but the loss is always 14,and don't convergence。Can you show me your training images?or tell me how to see them .I want to know whether my pictures are too complicated。Thanks.

**Comment by jonrein** [comment]
The inputs in the example code are processed features from speech waveforms, not images.  If you're trying to use this type of model structure with image data, you'll likely need a few more layers, with possibly a mix of convolution and LSTM layers.

**Comment by mrgloom** [comment]
Hello, can you specify how to prepair input and output for OCR like tasks?

Related:
https://github.com/igormq/ctc_tensorflow_example/issues/20
